source/manual/alerts/email-alert-api-high-queue-latency.html.md

```diff
@@ -5,19 +5,38 @@ section: Icinga alerts
 subsection: Email alerts
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-07-21
+last_reviewed_on: 2020-08-05
 review_in: 6 months
 ---
 
 ### High queue latency (queue_latency)
 
-This means the time it takes for a job to be processed is unusually high. This
-may indicate a problem down the line which is preventing workers from being
-processed. It may also imply the threshold is too low if a large number of
-emails have being sent out due to a content change.
+This means the time it takes for a job to be processed is unusually high. See
+our [Sidekiq queue latency] graph for more detail.
 
-See the [sidekiq][sidekiq] section for more information about the Sidekiq queues,
-or read [email troubleshooting].
+Some diagnostic steps you could take are below:
+
+* Check the [Sidekiq queue size] graph to see if we have a high number of emails
+  queued up.
+* If there is a high amount of emails in the queue, are we processing them at a
+  normal speed? Under high load we regularly send requests to Notify at a rate
+  of ~300 email requests per second. To check this, look at our [email send
+  attempts] graph.
+* Are the workers reporting any problems or any issues being raised in
+  [Sentry]?
+* Check [Kibana] to see our Sidekiq logs for Email Alert API.
+
+See the [Sidekiq] section for more information about Sidekiq or read
```

Comment:
We shouldn't link to generic steps unless we know they are helpful. In this case, the first two sections of the email troubleshooting are irrelevant.

Also, both these links are broken.

Reply:
I've removed the troubleshooting link. The links are made up of relative paths so are only broken when rendered by Github. I've made them absolute so you can click em here 😄